src/components/DeeplinkWrapper/deeplinkRoutes.js

```diff
@@ -21,6 +22,10 @@ export default [
     {
         // /v/*
         pattern: '/v($|(//*))',
+
+        // Disable deep linking in desktop App when passwordless is enabled because
+        // we want to open the magic link in its own tab
+        isDisabled: betas => Permissions.canUsePasswordlessLogins(betas),
```

Comment:
The `magic link` looks the same as the link used to `validate login `, which is going to be removed when Passwordless is fully rolled out in production. For now, we still want `validate login` to work in production for the users who don't have Passwordless beta enabled.

Reply:
It's fine to open the desktop App when Passwordless beta is not enabled, and a new account is created. The user clicks the link to validate the newly created account and this link looks exactly the same as the magic link. This is a temporary thing, because validate login will be removed when Passwordless is fully rolled out in ptoduction.

Reply:
> Or according to new logic, isn't it fine to open desktop app with magic link?

Oh, I think I missed your question. Yes, it is desired to _NOT_ have deep linking enabled when there's is the `magic code` flow, but we want the old flow (`validate login`) to stay unchanged, with deep linking enabled.